lldb/source/Plugins/TypeSystem/Clang/TypeSystemClang.cpp

```diff
@@ -4062,12 +4062,17 @@ TypeSystemClang::GetTypeInfo(lldb::opaque_compiler_type_t type,
     uint32_t vector_type_flags = eTypeHasChildren | eTypeIsVector;
```

Comment:
Is it worth testing for these too? If nothing else, it documents that a vector is marked as a vector and the type of its elements.

Reply:
Makes sense!